src/openrct2/world/Footpath.cpp

```diff
@@ -1163,9 +1164,10 @@ static bool neighbour_list_pop(rct_neighbour_list *neighbourList, rct_neighbour
     return true;
```

Comment:
the loop above (which I can't comment on) should be replaced with `memmove`

Reply:
The entire list could be replaced with a `deque`, that would also get rid of the hardcoded size limit. But I'd rather not touch it too much just before the release.